CompanyJobTitles.md

```diff
@@ -0,0 +1,58 @@
+Company Job Titles
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+The company Job Titles service provides a list of job titles (along with their ONet17 and Carotene 
+v3 classifications) that are personalized for a given company. Company Job Titles is 
```

Comment:
IMO the summary should just mention ONet and Carotene, and the taxonomy versions should be named in the Versioning section.

Reply:
Changed